src/main/java/com/flansmod/common/guns/InventoryHelper.java

```diff
@@ -18,10 +29,29 @@ public static boolean addItemStackToInventory(IInventory inventory, ItemStack st
             try {
                 int invSlot;
 
+                if(combineAmmo) {
+	                int numberBulletsLeft = stack.getMaxDamage()-stack.getItemDamage();
+	                
+	            	for(int i = 0; i < inventory.getSizeInventory()-4; i++) {
+	            		ItemStack is = inventory.getStackInSlot(i);
+	            		if(is == null || !is.isItemDamaged() || is.getItem() != stack.getItem()) continue;
+	            		
+	            		int itemDamageDiff = is.getItemDamage()-numberBulletsLeft;
+	            		if(itemDamageDiff >= 0) {
+	            			is.setItemDamage(itemDamageDiff);
+	            			return true;
+	            		} else {
+	            			is.setItemDamage(0);
+	            			numberBulletsLeft = Math.abs(itemDamageDiff);
```

Comment:
Why would it be necessary to have `Math.abs()` here?

Reply:
Wait... no. I set the numberBulletsLeft though.

Reply:
Ah true, you couldn't use a max then. But the abs still isn't needed

Reply:
Isn't that a bit confusing though to have numberBullets be a negative value?

Reply:
Sorry, had to go for a bit mid-comment and got confused. I'll have another look

Reply:
Yes, I see `numberBulletsLeft` is used in the next loop. And yeah you can't use max, I missed that.

But the code only enters else caluse if itemDamageDiff < 0, so Math.abs(itemDamageDiff) = -itemDamageDiff, so indeed Math.abs is unneeded. Just doing numberBulletsLeft = -itemDamageDiff is sufficient.

Idk if it would benefit from renaming the variables, but idk. Brain hurts. It's a minor thing, I'd only suggest removing the abs so it's more clear what's going on, otherwise it's fine.